app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/RpcInterceptor.kt

```diff
@@ -139,8 +140,10 @@ class RpcInterceptor(
 
         val errorMessage = serverError.error.data?.getErrorLog() ?: serverError.error.message
 
-        val errorType = serverError.error.data?.rpcErrorDetails
-        val domainErrorType = runCatching {
+        val errorType: JsonElement? = serverError.error.data
+            ?.rpcErrorDetails
+            ?.takeIf { !it.isJsonNull }
```

Comment:
error: null is parsed as JsonElement.Null and not JsonElement?

Reply:
json null value is the JsonElement (JsonNull)